### PR TITLE
Move from %v->%w for errs for PKI

### PR DESCRIPTION
**Pull request by cipherboy**
Also remove one duplicate error masked by return.

`Signed-off-by: Alexander Scheel <alex.scheel@hashicorp.com>`

### CODE REVIEW

File: builtin/logical/pki/crl_util.go [hunk 1]
```diff
@@ -196,7 +196,7 @@ func (cb *crlBuilder) checkForAutoRebuild(sc *storageContext) error {
 		// error.
 		defaultPeriod, defaultErr := time.ParseDuration(defaultCrlConfig.AutoRebuildGracePeriod)
 		if defaultErr != nil {
-			return fmt.Errorf("error checking for auto-rebuild status: unable to parse duration from both config's grace period (%v) and default grace period (%v):\n- config: %v\n- default: %v\n", cfg.AutoRebuildGracePeriod, defaultCrlConfig.AutoRebuildGracePeriod, err, defaultErr)
+			return fmt.Errorf("error checking for auto-rebuild status: unable to parse duration from both config's grace period (%v) and default grace period (%v):\n- config: %v\n- default: %w\n", cfg.AutoRebuildGracePeriod, defaultCrlConfig.AutoRebuildGracePeriod, err, defaultErr)
```
Comment (kitography) [inline builtin/logical/pki/crl_util.go:199]: I think there should be two %w here not one?  Both of those looks like errors to me.

**Review by kitography — APPROVED** [review]
This is awesome, thank you for doing this.

File: builtin/logical/pki/crl_util.go (hunk 1 above)
Comment (cipherboy) [reply builtin/logical/pki/crl_util.go:199]: Apparently you can only wrap one error with `fmt.Errorf`, not two. So I chose the last one to wrap in this place and one other (`pki/backend.go` -- auto-tidy and crl stuff in the `periodicFunc`).

Should both just be `%v` instead?

Comment (kitography) [reply builtin/logical/pki/crl_util.go:199]: *sigh*, no I think that's correct.  I'm sad at go here, not you (and absolutely didn't realize that).

Comment (cipherboy) [reply builtin/logical/pki/crl_util.go:199]: Cool, I'll merge then. Thank you!

**Pull request merged**